packages/extension/src/components/molecules/NetworkIndicator/NetworkIndicator.tsx

```diff
@@ -41,23 +51,24 @@ interface NetworkDisplayProps {
   description: string;
   isSelected?: boolean;
   onClick: () => void;
+  onRemove?: () => void;
 }
 
 const NetworkDisplay: FC<NetworkDisplayProps> = ({
```

Comment:
Maybe this component can be extracted here: `packages/extension/src/components/molecules/NetworkIndicator/NetworkDisplay/NetworkDisplay.tsx`.
This file starts to be quite big.

Reply:
Done